.gitpod.yml

```diff
@@ -0,0 +1,59 @@
+# Building pandas on init
+# Might delegate this later to prebuild with Q2 improvements on gitpod
+# https://www.gitpod.io/docs/config-start-tasks/#configuring-the-terminal
+# -------------------------------------------------------------------------
+
+# assuming we use dockerhub: name of the docker user, docker image, tag, e.g. https://hub.docker.com/r/pandas/pandas-gitpod/tags
+image:
+  file: pandas-dev/pandas-gitpod:latest
```

Comment:
Should this be updated?

Reply:
No. Once we have the automation, the latest tag will always have the latest image, and otherwise we will also have an archive of previous images with a date tag. See https://hub.docker.com/r/numpy/numpy-gitpod/tags. For now, I have been manually over-riding the latest tag with pushes from my local machine.

Reply:
Sorry, I wasn't very specific. I meant the "pandas-dev" org name (instead of "pythonpandas"), assuming this is the dockerhub org/image:version naming?